pkg/utils/utils.go

```diff
@@ -0,0 +1,16 @@
+package utils
+
+import (
+	"sigs.k8s.io/descheduler/pkg/api"
+)
+
+// GetNodeFitFromStrategyParams gets nodeFit configuration variable from the given StrategyParameters.
+// It will return false by default.
+func GetNodeFitFromStrategyParams(params *api.StrategyParameters) bool {
```

Comment:
Given the implementation of `GetNodeFitFromStrategyParams` is quite straightforward and the strategies already have `if strategy.Params != nil` clause, can you just inline the function? E.g.

```
	var nodeFit bool
	if strategy.Params != nil {
		...
		nodeFit = params.NodeFit
	}
```

Reply:
This is a fair point. Initially I had moved this into it's own function because I wanted to minimized the overhead required for all strategies to utilize the feature. Since the `GetNodeFitFromStrategyParams()` function now resides in it's own file, this seems a bit convoluted. I'll make it inline.